packages/moocfi-quizzes/src/utils/languages/english_options.ts

```diff
@@ -39,8 +39,7 @@ const englishLabels: SingleLanguageLabels = {
     submitPeerReviewLabel: "Submit review",
     peerReviewsCompletedInfo: "All peer reviews have been submitted",
     giveExtraPeerReviews:
-      "You have reviewed the minimum number of peer essays. You may continue to \
-        review your peers' works, thereby increasing the probability of your own answer being selected by others!",
+      "Give more peer reviews to have your answer revieved faster!",
```

Comment:
Should we still state that one doesn't have to give more reviews if they don't want to?